### PR TITLE
Load blog posts from API

### LINKED ISSUE #179: Study the possibility of displaying our blog posts on the News view
News tab currently displays the changelog to the left and a big static picture on the right.

Investigate the possibility of displaying instead information coming fom our blog API, currently living on:
https://changelog.unitystation.org/posts

There are many appraoches to this task: 
* It could simply show the ``socials_image`` and ``summary`` field of the post object, linking to the blog post on the website to read more about it.
* It could try to render the html content from the post object.
* If needed and estimated easier to do, I could add a new property to the post model with the content converted to markdown.

### PR DESCRIPTION
**Pull request by CorruptComputer**
Loads blog posts from the Unitystation API, unit tests included :)

Added this NuGet package for Avalonia, makes loading the images a lot easier: https://github.com/AvaloniaUtils/AsyncImageLoader.Avalonia

image

image

Fixes #179

### CODE REVIEW

**Comment by CorruptComputer** [comment]
Right now this just loads the latest 5 posts, but this can be easily adjusted if you would like to display more or less.

File: UnitystationLauncher.sln.DotSettings [hunk 1]
```diff
@@ -1,2 +1,4 @@
 ﻿<wpf:ResourceDictionary xml:space="preserve" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" xmlns:s="clr-namespace:System;assembly=mscorlib" xmlns:ss="urn:shemas-jetbrains-com:settings-storage-xaml" xmlns:wpf="http://schemas.microsoft.com/winfx/2006/xaml/presentation">
+	<s:Boolean x:Key="/Default/UserDictionary/Words/=Avalonia/@EntryIndexedValue">True</s:Boolean>
+	<s:Boolean x:Key="/Default/UserDictionary/Words/=Patreon/@EntryIndexedValue">True</s:Boolean>
 	<s:Boolean x:Key="/Default/UserDictionary/Words/=Unitystation/@EntryIndexedValue">True</s:Boolean></wpf:ResourceDictionary>
```
Comment (CorruptComputer) [inline UnitystationLauncher.sln.DotSettings:4]: These are just adding these to the spell check dictionary, I got tired of seeing those little blue underlines under these so I just added them.

**Comment by CorruptComputer** [comment]
Added the date that the blog post was published as well, this is what it looks like for now:

image

This probably needs someone with some artistic sense to style it up, but its at least functional for now.

**Comment by CorruptComputer** [comment]
It will show the title by default:
image

And the summary of the post when hovering:
image

**Review by corp-0 — APPROVED** [review]

**Pull request merged**